### PR TITLE
Small documentation improvements

### PR DESCRIPTION
**Pull request by nathanl**
- Show that `mix churn` is the basic command to run
- Improve the clarity of some sentences

### CODE REVIEW

**Comment by nathanl** [comment]
@patrykwozinski I had used https://github.com/chad/turbulence before and had been wishing a tool like that existed in Elixir. Bravo! 😀

File: README.md [hunk 1]
```diff
@@ -45,25 +49,27 @@ Example:
...
 Example
--C foo/bar.exs
+mix churn -c foo/bar.exs
 ```
 
 ## How to install
-The package can be installed by adding `churn` to your list of dependencies in `mix.exs` and then you need to copy `.churn.exs` [file](.churn.exs) into your project.
+
+The package can be installed by adding `churn` to your list of dependencies in `mix.exs` as follows.
+You also need to copy [`.churn.exs`](.churn.exs) into your project.
```
Comment (patrykwozinski) [inline README.md:72]: 👍🏻

File: README.md [hunk 2]
```diff
@@ -45,25 +49,27 @@ Example:
 --file-extensions [-e shortcut]
 
 Example
--e "ex,exs"
+mix churn -e "ex,exs"
```
Comment (patrykwozinski) [inline README.md:52]: I like this version of examples, good job!

File: README.md [hunk 3]
```diff
@@ -109,6 +117,19 @@ Copy and configure the `.churn.exs` file:
...
+Also, all scores produced by Churn are **relative to the project**.
+
+For example, in a newly-generated Mix project, with a simple `HelloWorld` module, that file will be scored `1.0` and be listed as the sole refactoring candidate.
+In any real project, the same module would have a very low score.
+
+On the other hand, in a project where every module is complex and frequently-modified, some modules may receive low scores which would receive high scores if they were part of a different project.
+
+When you see the results Churn produces, consider them the most likely refactoring candidates **for this project**.
+Whether they actually need refactoring is up to you.
+
```
Comment (patrykwozinski) [inline README.md:132]: Well done, thanks for that :)

**Review by patrykwozinski — APPROVED** [review]
Thanks a lot for contributing to this project! I'm so glad to see the first pull request opened by an open-source contributor. Welcome on board Nathan! :)

Great job 👏🏻

**Pull request merged**